compiler/vmdef.nim

```diff
@@ -169,6 +171,8 @@ type
     opcAsgnConst, # dest = copy(constants[Bx])
     opcLdGlobal,  # dest = globals[Bx]
     opcLdGlobalAddr, # dest = addr(globals[Bx])
+    opcLdGlobalDeref, # dest = globals[Bx][]
```

Comment:
These names need to reflect their FFI nature because I think for the general case this is unsound: You can alias globals via local pointers and then there is no ``global deref`` operation anymore and you better ensure the "local deref" is correct for all cases.

Reply:
done: 
opcLdGlobalDeref => opcLdGlobalDerefFFI
opcLdGlobalAddrDeref => opcLdGlobalAddrDerefFFI